cli_app/cli_codeable.rb

```diff
@@ -0,0 +1,82 @@
+require 'colorize'
+
+$task_array = []
+$id= 1
```

Comment:
HI! Why did you use global variable here?

Reply:
It's best to avoid declaring variables with a $ prefix, some special Ruby variables relating to system tasks are declared this way and we don't want to cause any confusion there :)